### PR TITLE
Ui live indicator

### PR DESCRIPTION
**Pull request by benjaminkay93**

### CODE REVIEW

**Review by moschopsuk — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,13 +1,15 @@
 {
 	"name": "la1tv-roses-graphics",
-	"version": "1.0.0",
+	"version": "2018.0.1",
```
Comment (stephenrob) [inline package.json]: When was the version change from SemVer ?

File: public/js/dashboard.js [hunk 2]
```diff
@@ -263,6 +289,22 @@ app.controller('lowerThirdsCGController', ['$scope', 'localStorageService', 'soc
 
         var stored = localStorageService.get('lower_thirds');
 
+        const showLiveLT = $scope => {
```
Comment (stephenrob) [inline public/js/dashboard.js]: Expanding `showLiveLT` to `showLiveLowerThird` etc helps with readability of code.

**Review by stephenrob — CHANGES_REQUESTED** [review]
Versioning in package.json needs to be discussed before merging.

File: package.json (hunk 1 above)
Comment (benjaminkay93) [reply package.json]: we werent semvering tbf, plus i feel it makes more sense to version on the year and minor/patching

File: public/js/dashboard.js (hunk 2 above)
Comment (benjaminkay93) [reply public/js/dashboard.js]: argeed will fix in a mo

File: package.json (hunk 1 above)
Comment (stephenrob) [reply package.json]: Lets not change the versioning scheme in this PR regarding UI live indicators we can switch to this at a later date

**Comment by benjaminkay93** [comment]
added netball aswell now its been pulled in

File: package.json (hunk 1 above)
Comment (benjaminkay93) [reply package.json]: fair play, think its worth changing though

**Comment by benjaminkay93** [comment]
changed back to semver now

**Review by stephenrob — APPROVED** [review]

**Pull request merged**